core/src/main/java/org/openmbee/sdvc/core/utils/RestUtils.java

```diff
@@ -33,4 +37,15 @@ public String getCookieValue(ResponseEntity<?> responseEntity, String cookieName
 
         return null;
     }
+    
+    
+    public HttpHeaders basicAuthHeader(String username, String password){
```

Comment:
This goes against our stateless architecture design goals. Why do we need to set cookies? If necessary, can we move this where it's utilized, instead of in core?

Reply:
This is just setting a basic auth header.  No cookies are being set

Reply:
Sorry, I was referring to the file as a whole, but it looks like you got it anyways. Thanks

Reply:
I don't think twc imports the auth module - seems like they're only used by twc anyways, may be better to just move to twc

Reply:
Ok, moved to twc module